common/lib/xmodule/xmodule/word_cloud_module.py

```diff
@@ -37,25 +37,25 @@ class WordCloudFields(object):
     """XFields for word cloud."""
     display_name = String(
         display_name=_("Display Name"),
-        help=_("Display name for this module"),
+        help=_("The label for the word cloud tool on the course page."),
```

Comment:
@clrux Sorry, could I make one more suggestion for this help text? "The label for this word cloud on the course page."
(it would refer to *this* word cloud, and it would not appear as a "tool")